The Go version of Dockerfile is old

## Problem

The following error occurs in `docker build .`.

```
# github.com/k0kubun/sqldef/adapter/postgres
adapter/postgres/postgres.go:82:16: undefined: strings.ReplaceAll
note: module requires Go 1.13make: *** [Makefile:22: deps] Error 2
The command '/bin/sh -c export GOPATH=/go/;   export GOBIN=$HOME/bin;   make all && sh -ec "mv build/*/mysqldef /usr/bin/ && mv build/*/psqldef /usr/bin/"' returned a non-zero code: 2
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,8 +1,8 @@
-FROM golang:1.11.11-alpine3.8
+FROM golang:1.15.6-alpine3.12
 RUN apk add --no-cache build-base make gcc git
 
 COPY . /sqldef
 WORKDIR /sqldef
 RUN export GOPATH=/go/; \
   export GOBIN=$HOME/bin; \
-  make all && sh -ec "mv build/*/mysqldef /usr/bin/ && mv build/*/psqldef /usr/bin/"
+  make all && sh -ec "mv build/*/mysqldef /usr/bin/ && mv build/*/psqldef /usr/bin/ && mv build/*/sqlite3def /usr/bin/"

